### DISCUSSION TITLE
usecases/segmentation-masks-detect-sam2/

### DISCUSSION THREAD

**Discussion by giscus** [RizwanMunawar/visionusecases#2] [Community]
# usecases/segmentation-masks-detect-sam2/

Learn how to generate segmentation masks with object detection and SAM2 models for advanced image processing tasks.

https://visionusecases.com/usecases/segmentation-masks-detect-sam2/

**Comment by RizwanMunawar** [comment]
I ❤️ contributing to open-source! If you have an awesome computer vision project you'd like to showcase here, feel free to open a PR by following the contribution guidelines: https://github.com/RizwanMunawar/visionusecases?tab=readme-ov-file#how-to-contribute-  

It doesn't matter, If the project is small, your contributions will be accepted :)

This website is built by the community, for the community ❤️.
